src/styles/App/_events-by-category-page.scss

```diff
@@ -0,0 +1,87 @@
+.events-by-category-page {
+  > h2 {
+    font-size: 2.3rem;
+  }
+  .events-list {
+    .event-card {
+      background-color: #fff;
+      width: calc(70% - 30px);
+      height: 190px;
+      list-style: none;
```

Comment:
try not to add an height, but let the elements define the height for you! it would be easier to make it responsive!